README.md

```diff
@@ -113,59 +113,4 @@ in a watched mode with growl turned on.
 
 Refer to build commands section to look up the commands to open the StoryTeller editor or run the StoryTeller specs.
 
-## Documentation
```

Comment:
@oskardudycz Looks like the PR also brought in changes to README, can you please check?

Reply:
Nice catch, Babu! I don't know how did it get there o_O Fixed!